srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -17,6 +33,146 @@
     
+    private List<Specialty> applicableSpecialties;
+    
+    private static final Logger fLogger = LoggerFactory.getLogger(EditRiskModel.class);
+
+    /**
+     * Class to hold the label, type and coefficient for a (link @ModelTerm) 
+     * and its coefficient on the Admin's Edit Model page. 
+     * 
+     * @author owner
+     *
+     */
+    public static class ModelTermSummary  implements Comparable<ModelTermSummary>
+    {
+        private final ModelTerm targetTerm;
+        
+        private String displayName;   // the name presented on the Edit Model page
+        private String termType;      // the variable type
+        
+        private ModelTermSummary( ModelTerm mt, String name, String type ) 
+        {
+            displayName = name;
+            targetTerm = mt;
+            termType = type;
+        }
+         
+        public String getDisplayName()  
+        {
+            return displayName;
+        }
+        
+        public String getTermType() 
+        {
+            return termType;
+        }
+        
+        public float getCoefficient()
+        {
+            return targetTerm.getCoefficient();
+        }
+
+        public static ModelTermSummary createTermSummary( ModelTerm modelTerm ) 
+        {
+            String dispName="";
+            String type="";
+            
+            if( modelTerm instanceof ConstantTerm ) 
+            {
+                dispName = "Constant"; 
+                type = "Constant"; // or N/A or blank
+            }
+            else if( modelTerm instanceof ProcedureTerm )
+            {
+                type = "Multiplier"; // ???
+                dispName = "Procedure (RVU Multiplier)"; 
+            }
+            else if( modelTerm instanceof BooleanTerm )
+            {
+                type = "Boolean";
+                BooleanTerm boolTerm = (BooleanTerm)modelTerm;                
+                dispName = boolTerm.getVariable().getDisplayName();
+            }            
+            else if( modelTerm instanceof NumericalTerm )
+            {
+                type = "Numerical";
+                NumericalTerm numTerm = (NumericalTerm)modelTerm;
+                dispName = numTerm.getVariable().getDisplayName();                                     
+            }
+            else if( modelTerm instanceof DiscreteTerm )
+            {
+                DiscreteTerm discreteTerm = (DiscreteTerm)modelTerm;                                
+                MultiSelectOption opt = discreteTerm.getOption();
+                DiscreteVariable var = discreteTerm.getVariable();
+                
+                if( var instanceof DiscreteNumericalVariable ) 
+                {
+                    type = "Discrete Numeric";
+                    dispName = var.getDisplayName()+ " = " + opt.getValue();
+                }
+                else if( var instanceof MultiSelectVariable ) 
+                {
+                    type = "Discrete";
```

Comment:
Ok I'll change it to be consistent with the Variable type. We should ask them what they want to call it.   I still think this is not  a good term/word  to use.  The word 'multi-select' doesn't specify a type. Its a selection process where multiple selections can be made at once.